src/loader/x86_64/elf/mod.rs

```diff
@@ -86,6 +86,12 @@ impl error::Error for Error {
     }
 }
 
+impl Display for Error {
+    fn fmt(&self, f: &mut fmt::Formatter) -> fmt::Result {
+        write!(f, "Kernel Loader Error: {}", self.description())
```

Comment:
`description` is deprecated in newer rust versions. See #35

Reply:
Yes, both description() and cause() are deprecated in favor of Display/source().
I have just moved the position of the code without any changes:)

Reply:
Ok.